src/main/java/uk/gov/register/serialization/handlers/AddItemCommandHandler.java

```diff
@@ -17,7 +18,7 @@ public AddItemCommandHandler() {
     }
 
     @Override
-    protected void executeCommand(RegisterCommand command, Register register) {
+    protected void executeCommand(RegisterCommand command, Register register, ProofGenerator _proofGenerator) {
```

Comment:
I guess we are trying to indicate an unused argument here with the underscore? In Java would just stick with `camelCase` even for an unused override variable like this.